src/Scene/Node.js

```diff
@@ -24,6 +24,7 @@ define('Scene/Node', [], function() {
         this.screenSpaceError = 0.0;
         this.loaded = false;
         this.pendingSubdivision = false;
+        this.pendingLayers = {};
```

Comment:
I think we should support this behaviour, we may want to send the commands for all the layers concurrently in the future.